doc/developers/maintainer.rst

```diff
@@ -1,14 +1,33 @@
 Maintainer / core-developer information
 ========================================
 
+A major release, incrementing the minor version, or a bug fix release
+incrementing the patch version. Our convention is that we release one or more
+release candidates (0.RRrcN) before releasing the final distributions. Creating
+the first RC involves creating the release branch (0.99.X for instance)
+directly on the main repo. This is considered the *feature freeze*. The rest of
+the changes (required to fix some of the issues detected during the release
+process are a good example) should be done as a PR to the master branch, if
+it's possible, or done through a PR to the release branch, for the other
+maintainers to review before the release. Note that all development for the
+minor releases and the major release of for instance 0.99, happens under the
+branch called ``0.99.X``, where ``X`` is a literal, and independent of minor
+bug fix releases. Each release candidate, major, or minor release is a tag
```

Comment:
I don't understand this sentence, especially since it is said above that changes should be done to master whenever possible

Reply:
The development happens on that branch, through PRs which are merged into that branch.